addons/account/models/account_move.py

```diff
@@ -2139,13 +2140,6 @@ def post(self):
                     to_write['line_ids'].append((1, line.id, {'name': to_write['invoice_payment_ref']}))
                 move.write(to_write)
 
-            if move == move.company_id.account_opening_move_id and not move.company_id.account_bank_reconciliation_start:
-                # For opening moves, we set the reconciliation date threshold
-                # to the move's date if it wasn't already set (we don't want
-                # to have to reconcile all the older payments -made before
-                # installing Accounting- with bank statements)
-                move.company_id.account_bank_reconciliation_start = move.date
-
```

Comment:
removing the field account_bank_reconciliation_start implies that we have to have a solution for people that start using invoicing now, and later on switch to the full accounting features.

Not sure what you had in mind is good enough

Reply:
Without the `group_account_user` group, the payments are created directly  using the bank account and then, doesn't have to be matched with a statement line. Then, you only need to make a statement line for payments created with the full accounting feature.